base_exception/tests/test_base_exception.py

```diff
@@ -2,43 +2,116 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from odoo.tests import common
+from odoo import api, fields, models
+from .common import init_test_models
 
 import logging
 
 _logger = logging.getLogger(__name__)
 
 
-# @common.at_install(False)
-# @common.post_install(True)
-class TestBaseException(common.TransactionCase):
+class TestBaseException(common.SavepointCase):
 
-    def setUp(self):
-        super(TestBaseException, self).setUp()
+    # pylint: disable=missing-return
 
-        self.base_exception = self.env['base.exception']
-        self.exception_rule = self.env['exception.rule']
-        self.exception_confirm = self.env['exception.rule.confirm']
+    @classmethod
+    def setUpClass(cls):
+        super(TestBaseException, cls).setUpClass()
 
-        self.exception_rule._fields['rule_group'].selection.append(
+        class PurchaseTest(models.Model):
```

Comment:
I'd move this to a separated file anyway and here run only `init_test_models`

Reply:
Done.